src/components/CheckIn/AddPieceModal.js

```diff
@@ -139,6 +174,7 @@ class AddPieceModal extends Component {
 }
 
 export default stripesForm({
-  form: 'AddPieceModalForm',
+  enableReinitialize: true,
```

Comment:
could you please describe the reason of using this? thanks

Reply:
Checked it. No needed